### DISCUSSION TITLE
Parcel v2 "--global" replacement

### DISCUSSION THREAD

**Discussion by progrAmer** [parcel-bundler/parcel#6437] [General] [answered]
Is there a plan, in the near future, to offer replacement for "--global" flag?

**Comment by DeMoorJasper** [comment]
Currently we're focusing on getting a release candidate out and a stable version. But this seems like a feature we will definitely support eventually, feel free to try and implement this.

**Reply by oshliaer** [reply]
@DeMoorJasper 

```sh
❯  npx parcel --version
2.0.0
```

I see `globalName` in my bundle and it works as I expect. But I can't pass it to the IIFE of the bundle via cli.
![1635827570238](https://user-images.githubusercontent.com/5027939/139787352-bd4f873a-4a28-4bfc-8524-3c3f48f6a6de.png)

Is that what you are talking about?
Can I pass the name to cli?

**Comment by danmarshall** [comment, marked as answer]
Related: #5583
